api/src/utils/put-utils.js

```diff
@@ -170,13 +170,39 @@ exports.editRecordWithFlavours = async function (args, res, next, incomingObj, e
         entry[0].includes('BCMI') && (incomingObj.isBcmiPublished = false);
       }
 
-      if (flavourIncomingObj[entry[0]]._id) {
+      const flavourId = flavourIncomingObj[entry[0]]._id;
+
+      if (flavourId) {
+        // Grab the existing flavour object to get the issuingAgency and issuedTo if
+        // they don't exist in the flavourIncomingObj
+        const flavourRecordModel = mongoose.model(entry[0]);
+
+        const flavourExistingObject = await flavourRecordModel
+          .findOne({
+            _id: new ObjectId(flavourId)
+          });
+
+        const flavourExistingJSONObject = flavourExistingObject.toJSON();
```

Comment:
Could just use `flavourExistingObject` itself without doing `.toJSON()`

Reply:
I think this throws an error in processPutRequest in the record controller because the issuedTo isn't a json object, but like a mongoose object or something. Maybe there is a better way to handle this though?